### ISSUE TITLE
Buttons on Scrape /targets page do not have Bootstrap framework style

### ISSUE THREAD

**Issue by denisgolius** [VictoriaMetrics/VictoriaMetrics#1422]
**Describe the bug**
Buttons on Scrape /targets page do not have Bootstrap framework style
![image](https://user-images.githubusercontent.com/5650611/124025498-2c678d00-d9f9-11eb-94d1-cc124d5244f9.png)

**To Reproduce**

For Victoria Metrics Single-Node
```bash
git clone https://github.com/VictoriaMetrics/VictoriaMetrics.git
cd VictoriaMetrics/deployment/docker
docker-compose up -d
```
open http://127.0.0.1:8428/targets in browser and see results.

For Victoria Metrics Cluster
```bash
git clone https://github.com/VictoriaMetrics/VictoriaMetrics.git
cd VictoriaMetrics/deployment/docker
git checkout cluster
docker-compose up -d
```
open http://127.0.0.1:8429/targets in browser and see results.

**Expected behavior**
Buttons are displayed with the applied style from Bootstrap
![image](https://user-images.githubusercontent.com/5650611/124024805-5e2c2400-d9f8-11eb-9bbe-74e8e20d9be0.png)

**Version**
This problem only appears in version 1.62.0 in both Single and Cluster versions. For version 1.61.0 and 1.61.1 it works properly.

**Comment by valyala** [comment]
The issue has been fixed in the commit ae485c2bfd789d64c717ee7ec1795a210f913ada . This commit will be included in the next release.

**Comment by valyala** [comment]
The bugfix has been included in [v1.63.0](https://github.com/VictoriaMetrics/VictoriaMetrics/releases). Closing the issue as fixed.
